hack/lib/init.sh

```diff
@@ -44,6 +44,31 @@ source "${KUBE_ROOT}/hack/lib/etcd.sh"
 
 KUBE_OUTPUT_HOSTBIN="${KUBE_OUTPUT_BINPATH}/$(kube::util::host_platform)"
 
+# list of all available group versions.  This should be used when generated code
+# or when starting an API server that you want to have everything.
+# most preferred version for a group should appear first
+KUBE_AVAILABLE_GROUP_VERSIONS=${KUBE_AVAILABLE_GROUP_VERSIONS:-"\
```

Comment:
I think you're getting literal quotes in this list, and the slashes are unnecessary:
```sh
KUBE_AVAILABLE_GROUP_VERSIONS="${KUBE_AVAILABLE_GROUP_VERSIONS:-my
list
of
things}"
```

Also, this *screams* `array` to me, but since the GVs can't contain spaces I think it's a moot point.